doc/articles/features/using-il-linker-webassembly.md

```diff
@@ -1,4 +1,26 @@
-# Using the IL Linker for WebAssembly
+---
+uid: articles.features.illinker
+---
+
+# Using the IL Linker
```

Comment:
Is this needed ?

Reply:
If yes, should we rename the file ?

Reply:
The problem is SEO related, and I wonder if we should change the name...